packages/venia-ui/lib/components/OrderHistoryPage/orderHistoryPage.gql.js

```diff
@@ -14,25 +14,86 @@ export const GET_CUSTOMER_ORDERS = gql`
             id
             orders @client {
                 items {
+                    billing_address {
+                        city
+                        country_code
+                        firstname
+                        lastname
+                        postcode
+                        region_id
+                        street
+                        telephone
+                    }
                     id
                     invoices {
                         id
                     }
                     items {
                         id
+                        product_name
+                        product_sale_price
                         product_sku
+                        selected_options {
+                            label
+                            value
+                        }
+                        quantity_ordered
                     }
                     number
                     order_date
+                    payment_methods {
+                        name
+                        type
+                        additional_data {
+                            # additional data is not a constant documented in the schema architecture doc because
+                            # it changes based on the type of payment method. This schema I have here is temp based
+                            # on what braintree returns in checkout step. Once orders GQL coverage is ready, we have to revist this.
+                            card_type
+                            last_four
+                        }
```

Comment:
Are you saying that this is not actually going to exist? Or are you saying that it _could_ exist if you have braintree enabled? I'm a little confused about the comment.

Reply:
Sorry, this is an old comment I forgot to remove. It is not needed anymore.